bsb/morphologies/__init__.py

```diff
@@ -1131,10 +1131,13 @@ def versor(self):
         """
         Return the normalized vector of the axis connecting the start and terminal points.
         """
-        try:
-            return (self.end - self.start) / np.linalg.norm(self.end - self.start)
-        except IndexError:
-            raise EmptyBranchError("Empty branch has no versor") from None
+        versor = (self.end - self.start) / np.linalg.norm(self.end - self.start)
+        if np.any(np.isnan(versor)):
+            raise EmptyBranchError(
+                "Empty and single-point branched have no versor"
+            ) from None
```

Comment:
```suggestion
            )
```

'from None` is only required inside of `except` clauses while handling another error.